File: jesus_sotomayor_rodriguez/Programas/dart/Mayo2024/asknumber.dart
import 'dart:io';
void main(){
  print('ingrese un número');
  int num1 = int.parse(stdin.readLineSync()!);

  print('ingrese un segundo número');
  int num2 = int.parse(stdin.readLineSync()!);

  int suma = num1 + num2;
  int resta = num1 - num2;
  int mult = num1 * num2;
  double div = num1 / num2;

  print('El resultado de la suma de los números es: $suma');
  print('El resultado de la resta de los números es: $resta');
  print('El resultado de la multiplicacion de los números es: $mult');
  print('El resultado de la division de los números es: $div');
}

File: jesus_sotomayor_rodriguez/Programas/ruby/Mayo2024/asknumber.rb
puts "Por favor introduce un número:"
num1 = gets.chomp.to_i

puts "Por favor introduce un segundo número:"
num2 = gets.chomp.to_i

suma = num1 + num2
resta = num1 - num2
mult = num1 * num2
div = num1.to_f / num2

puts "El resultado de la suma de los números: #{suma}"
puts "El resultado de la resta de los números: #{resta}"
puts "El resultado de la multiplicacion de los números: #{mult}"
puts "El resultado de la division de los números: #{div}"